web/global.d.ts

```
@@ -10,11 +10,25 @@ declare global {
     export default path;
   }
+  interface Window {
+    google: {
+      translate: {
+        TranslateElement: new (
+          config: {
+            pageLanguage: string;
+            includedLanguages: string;
+          },
+          elementId: string
+        ) => void;
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)